File: src/main/java/ru/ioleksiv/telegram/bot/api/model/annotation/validator/impl/PasportDataFilterValidator.java
package ru.ioleksiv.telegram.bot.api.model.annotation.validator.impl;

import ru.ioleksiv.telegram.bot.api.model.annotation.validator.CustomValidator;
import ru.ioleksiv.telegram.bot.api.model.annotation.validator.FilterValidator;
import ru.ioleksiv.telegram.bot.api.model.objects.passport.PassportData;

public class PasportDataFilterValidator extends FilterValidator<PassportData> {

    public PasportDataFilterValidator(CustomValidator<PassportData> validator) {
        super(PassportData.class, validator);
    }
}

File: core/src/main/kotlin/io/github/oleksivio/tl/kbot/core/model/annotation/validator/impl/PasportDataFilterValidator.kt
package io.github.oleksivio.tl.kbot.core.model.annotation.validator.impl

import io.github.oleksivio.tl.kbot.core.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.passport.PassportData
import io.github.oleksivio.tl.kbot.core.core.controller.handler.check.Validator

class PasportDataFilterValidator(validator: Validator<PassportData>) : FilterValidator<PassportData>(
    PassportData::class, validator)
